tasties_app/tests/test_rating.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from django.contrib.auth.models import User
+
+from tasties_app.models import (
+    Rating,
+    Recipe,
+)
+
+
+@pytest.fixture()
+def generate_data():
+    """
+    This fixture create the data we need in order to execute the tests.
+    """
+    user1 = User.objects.create_user("testuser1", "password")
+    recipe = Recipe(
+        title="Test Recipe1",
+        author_id=user1,
+        description="Test Description1",
+        directions="Test Directions1",
+        minutes_to_make=1,
+        recipe_picture="test_picture2",
+    )
+    recipe.save()
+
+    rating = Rating(author_id=user1, recipe_id=recipe, rating=3)
+    rating.save()
+
+
+class TestRatingModel:
+    @pytest.mark.django_db
+    def test_add_rating(self, generate_data):
+        rating_set = Rating.objects.all()
+        rating = rating_set[0]
+        assert rating.rating == 3
+
+    @pytest.mark.django_db
+    def test_delete(self, generate_data):
```

Comment:
Please give the test a meaningful name, something like this
```suggestion
    def test_delete_rating(self, rating):
```